processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -104,6 +104,8 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
+import static io.druid.query.groupby.GroupByQueryRunnerTestHelper.*;
```

Comment:
we try not to do import *s anywhere. Most IDEs shoudl be able to auto include the files you require